proving_system/plonk/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub mod prover;
+pub mod setup;
+pub mod test_utils;
+pub mod verifier;
```

Comment:
Test utils are only used in tests, so we don't need to compile it in release mode

```suggestion
pub mod verifier;
#[cfg(test)]
pub mod test_utils;
```